### PR TITLE
Emphasis of LoadOlderEmail (fixing #168)

### PR DESCRIPTION
**Pull request by EightSQ**
We added a gray tone as background color for the LoadOlderEmailButton Email.

Result

---
This change is [Reviewable](https://reviewable.io/reviews/hpi-swa-teaching/imapclient/176)

### CODE REVIEW

**Comment by Paulpanther** [comment]
Might look bad on different themes, since the color is hardcoded

**Comment by EightSQ** [comment]
> Might look bad on different themes, since the color is hardcoded

@Paulpanther 3244d2c implements color selection based on the current (image) theme.

File: packages/IMAPClient-UI.package/ICFolderDialog.class/instance/buildWith..st [hunk 1]
```diff
@@ -83,6 +83,15 @@ buildWith: builder
 		autoDeselect: false;
 		columns: {
 			[:listMorph | self emphasizeUnseenMessages: listMorph filteredItems.
+				listMorph filteredItems
```
Comment (felixrindt) [inline packages/IMAPClient-UI.package/ICFolderDialog.class/instance/buildWith..st]: Why not move this into it's own method like in line 85?

Comment (EightSQ) [reply packages/IMAPClient-UI.package/ICFolderDialog.class/instance/buildWith..st]: Done in 86d2b31.

**Review by felixrindt — APPROVED** [review]

**Review by timgarrels — APPROVED** [review]

**Pull request merged**